CHANGELOG.rst

```diff
@@ -35,6 +35,8 @@ Improved Documentation
   xdist could identify the wrong test as failing when test crashes at teardown.
   To address this issue, xdist now marks test as complete at teardown.
 
+- 168.trivial: add long option `--numprocesses` as alternative for `-n`.
```

Comment:
This should be considered a "new change" because it will require an increase in the minor version (so people that depend on using `--numprocesses` can depend on a "minor" version and not a "patch" version.

Also, this should be a new file in the `changelog` folder named `168.feature` with the contents you wrote:

```
Add long option `--numprocesses` as alternative for `-n`.
```